app/views/govuk_publishing_components/components/_translation-nav.html.erb

```diff
@@ -2,11 +2,19 @@
   translations ||= []
   brand ||= false
   brand_helper = GovukPublishingComponents::Presenters::BrandHelper.new(brand)
+
+  def tracking_is_present(translations)
```

Comment:
Can we move this into a helper to tidy up the view?

Reply:
@vanitabarrett done